readme.md

```diff
@@ -86,9 +86,22 @@ const honey = useBearStore((state) => state.honey)
 
 If you want to construct a single object with multiple state-picks inside, similar to redux's mapStateToProps, you can tell zustand that you want the object to be diffed shallowly by passing the `shallow` equality function.
 
+To use a custom equality function, you need `createWithEqualityFn` from `zustand/traditional`. It takes the second argument for the default equality function.
+
 ```jsx
+import { createWithEqualityFn } from 'zustand/traditional'
 import { shallow } from 'zustand/shallow'
 
+// Use createWithEqualityFn instead of create
+const useBearStore = createWithEqualityFn(
+  (set) => ({
+    bears: 0,
+    increasePopulation: () => set((state) => ({ bears: state.bears + 1 })),
+    removeAllBears: () => set({ bears: 0 }),
+  }),
+  Object.is // Specify the default equality function, which can be shallow
```

Comment:
If the eq fn is shallow, what is the effect of using 'zustand/shallow'? Isn't the eq fn always shallow then, regardless of it?

Reply:
If you do this:
```js
import { createWithEqualityFn } from 'zustand/traditional'
import { shallow } from 'zustand/shallow'

const useMyStore = createWithEqualityFn(..., shallow)
```
You don't need to specify shallow for useMyStore:
```js
  useMyStore(...)
  // that 👆 is the same as this 👇 
  useMyStore(..., shallow)
```

Reply:
```suggestion
  customEqualityFunction // This will become the default equality function, unless you pass another one to `useBearStore` hook.
```

And then I'd add a `customEqualityFunction`.